### ISSUE TITLE
Sandboxie-plus 1.3.3: Warnings at each Start of SandMan.exe that only Admins are allowed to make changes

### ISSUE THREAD

**Issue by VisionXi** [sandboxie-plus/Sandboxie#2237]
### Describe what you noticed and did

If user has no admin rights and you have configured "Config protection" -> "Only Administrator user accounts can make changes", then with each start of SandMan.exe (that is, with each login) you get the warning pop up that only admin users can make changes.
This annoying behavior came with update 1.3.2 -> 1.3.3
Work around is of course to do without this config protection, but it's a good feature and I WANT TO USE IT AGAIN :)
I see this bahavior unter several installations of Windows 11 und Windows 10.

### How often did you encounter it so far?

Every time SandMan.exe starts

### Affected program

Independent from application

### Download link

Not relevant

### Where is the program located?

Not relevant to my request.

### Expected behavior

No warning when "Config protection" -> "Only Administrator user accounts can make changes" is configured and a normal user without admin rights starts Sandman.exe. Like is was before the update 1.3.2 -> 1.3.3

### What is your Windows edition and version?

Windows 10 and Windows 11, several installations

### In which Windows account you have this problem?

Not relevant to my request.

### Please mention any installed security software

Only Microsoft Defender, but this doesn't matter.

### What version of Sandboxie are you running?

Sandboxie Plus 1.3.3

### Is it a new installation of Sandboxie?

I recently did a new clean installation.

### Is it a regression?

Yes. It came with the update from 1.3.2 to 1.3.3

### In which sandbox type you have this problem?

Not relevant to my request.

### Can you reproduce this problem on an empty sandbox?

I can confirm it also on an empty sandbox.

### Did you previously enable some security policy settings outside Sandboxie?

No

### Sandboxie.ini configuration

```shell
Does not matter. I can reproduce this with a clean installation without an already existing Sandboxie.ini
```

**Comment by Luro223** [comment]
@VisionXi it is expected behaviour UPDATE 1.3.3:`Changed
improved SandMan settings behaviour for non admin users https://github.com/sandboxie-plus/Sandboxie/issues/2123`

**Comment by APMichael** [comment]
This is a side effect of this issue: #2195
(Should be finally fixed with the next version.)

**Comment by DavidXanatos** [comment]
As already noted this message is a side effect of https://github.com/sandboxie-plus/Sandboxie/issues/2195 which will be fixed in the next build

**Comment by VisionXi** [comment]
Yes, also from my side as the initiator of this issue, - can be closed. Fixed with 1.3.4.
Thank you!
